client/blocks/plugin-scheduled-updates-common/error-utils.tsx

```diff
@@ -0,0 +1,27 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import type { APIError } from '@automattic/data-stores';
+
+export const handleErrorMessage = ( error: APIError ): string => {
+	let errorMsg = error.message;
+	if ( error.status === 500 ) {
```

Comment:
What do you think about catching errors by status wrapping it with `switch/case`.
The default could be `error.message`.

By this, we prepare the util function to add future error message handlers easily.

Reply:
Nice suggestion, thanks!